docs/production.md

```diff
@@ -0,0 +1,101 @@
+# Running KUDO Cassandra in production
+
+Before running KUDO Cassandra in production please follow this guide to ensure
+the reliable stability of your production cluster.
+
+And also read the
+[Cassandra Anti-Patterns](https://docs.datastax.com/en/dse-planning/doc/planning/planningAntiPatterns.html)
+guidelines to not to follow any anti-patterns practices when running production
+workload.
+
+## Compute Resources
+
+For production use-case of KUDO Cassandra we recommend a minimum of 32 GiB of
+memory and 16 cores of CPU for the guaranteed stability.
+
+Read
+[Capacity Planning](https://docs.datastax.com/en/dse-planning/doc/planning/capacityPlanning.html)
+around how to do capacity planning for a Cassandra installation.
+
+## Storage
+
+Verify if there is a storage class installed in the Kubernetes cluster. In this
+example we will use the `aws-ebs-csi-driver` as the storage class reference.
+
+```
+$ kubectl get sc
+NAME                             PROVISIONER       AGE
+awsebscsiprovisioner (default)   ebs.csi.aws.com   2d
+```
+
+### Volume Expansion
+
+Verify if the storage class has the option `AllowVolumeExpansion` and is set to
+`true`.
+
+```
+$ kubectl describe sc awsebscsiprovisioner
+Name:                  awsebscsiprovisioner
+IsDefaultClass:        Yes
+Annotations:           kubernetes.io/description=AWS EBS CSI provisioner StorageClass,storageclass.kubernetes.io/is-default-class=true
+Provisioner:           ebs.csi.aws.com
+Parameters:            type=gp2
+AllowVolumeExpansion:  true
+MountOptions:          <none>
+ReclaimPolicy:         Delete
+VolumeBindingMode:     WaitForFirstConsumer
+Events:                <none>
+```
+
+:warning: In case `AllowVolumeExpansion` is `unset` or `false`, make sure to
+provision enough disk when bootstrapping the KUDO Cassandra cluster. The disk
+size can be configured using the `DISK_SIZE` parameter. By **default, DISK_SIZE
+is set to 20Gi** and is **not ideal for production usage**. Users should
+increase disk size by as much as they deem necessary for reliable stability.
+
+### ReclaimPolicy
+
+Verify the storage class has the option `ReclaimPolicy` and is set to `Retain`.
+
+To read more about the `ReclaimPolicy` read the official Kubernetes docs on
+[Changing the Reclaim Policy](https://kubernetes.io/docs/tasks/administer-cluster/change-pv-reclaim-policy/)
+
+```
```

Comment:
```suggestion
```

Reply:
code won't wrap so if this is supposed to be a block quote, it should only have `> ` preceding each line.